python/tests/hypothesis/arcticdb/test_hypothesis_version_store.py

```diff
@@ -313,14 +314,15 @@ def test_batch_read_snapshot(self, snap: str):
 
 
 @pytest.mark.parametrize("lib_type", ["lmdb_version_store_delayed_deletes_v1", "lmdb_version_store_delayed_deletes_v2"])
+@SLOW_TESTS_MARK
 def test_stateful(lib_type, request):
     VersionStoreComparison._lib = request.getfixturevalue(lib_type)
     run_state_machine_as_test(
         VersionStoreComparison,
         settings=settings(  # Note: timeout is a legacy parameter
-            max_examples=int(os.getenv("HYPOTHESIS_EXAMPLES", 100)),
+            max_examples=int(os.getenv("HYPOTHESIS_EXAMPLES", 10)),
             deadline=None,
-            stateful_step_count=100,
+            stateful_step_count=10,
```

Comment:
10 is really too low. This test is designed to spend a few minutes exercising the code.

The last time I touched it, the test should finish in about 2 minutes under the default settings. If that's no longer the case, then we had a performance regression.